src/Microsoft.AspNetCore.Mvc.Testing/build/netstandard2.0/Microsoft.AspNetCore.Mvc.Testing.targets

```diff
@@ -15,11 +15,38 @@
     <PreserveCompilationContext>true</PreserveCompilationContext>
   </PropertyGroup>
 
-  <Target Name="CopyAditionalFiles" AfterTargets="Build" Condition="'$(TargetFramework)'!=''">
+    <Target Name="_ResolveMvcTestProjectReferences" DependsOnTargets="ResolveReferences">
+      <ItemGroup>
+        <_ContentRootProjectReferences Include="@(ReferencePath)" Condition="'%(ReferencePath.ReferenceSourceTarget)' == 'ProjectReference'" />
+      </ItemGroup>
+    </Target>
+
+  <Target Name="_AddContentRootForProjectReferences" BeforeTargets="BeforeCompile" DependsOnTargets="_ResolveMvcTestProjectReferences">
+    <ItemGroup>
+      <WebApplicationFactoryContentRootAttribute 
```

Comment:
You might need a condition here because batching always runs at least once. 
`Condition=" '%(_ContentRootProjectReferences.Identity)' != '' "`